htmresearch/frameworks/pytorch/sparse_mnist_cnn.py

```diff
@@ -71,7 +71,8 @@ def __init__(self,
                c1OutChannels=20,
                c1k=20,
                n=50,
-               useDropout=True,
+               k=50,
```

Comment:
Missing doc for `k`. I am assuming it is the same as `l2k`

Reply:
Thanks - fixed the docstring.